### ISSUE TITLE
Test bw_better_autop() with WordPress 4.7

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-css#3]
During testing of WordPress 4.7-beta3 ( & 4 ) the following message was displayed when WP_DEBUG is true.

`Notice: Indirect modification of overloaded element of WP_Hook has no effect in C:\apache\htdocs\wordpress\wp-content\plugins\oik\includes\oik-filters.inc on line 67`

oik-css uses the `bw_replace_filter()` function from oik's `includes/oik-filters.inc`
This function sees `$wp_filters` as a multi-dimensional array. 

The Notice is issued from
```
    $saved =& $wp_filter[$tag][$priority][$function_to_replace];
```

In WordPress 4.7 the $wp_filter array is of WP_Hook objects.

This issue is associated with another issue in the oik base plugin. https://github.com/bobbingwide/oik/issues/58

**Comment by bobbingwide** [comment]
oik-css does not need to be changed, but a new version of oik will be needed for WordPress 4.7

**Comment by bobbingwide** [comment]
Oik-css v0.9.0 will be dependent upon oik v3.2.3. Also it will released as Requires 4.9 and tested up to 4.9.1. There are some basic tests for bw_better_autop() implemented in the tests for the [bw_autop] shortcode.
